include/NovelRT/Maths/GeoVector2F.h

```diff
@@ -21,157 +24,789 @@ namespace NovelRT::Maths
+         */
         inline GeoVector2F operator*(float other) const noexcept
         {
             return GeoVector2F(*reinterpret_cast<const glm::vec2*>(this) * other);
         }
 
+        /**
+         * @brief Constructs a GeoVector2F containing the quotents of a Hadamard division using this and another scalar.
+         *
+         * @details
+         * An example of a division between a vector and a scalar: \f[
+         *      \begin{pmatrix}
+         *      16\\
+         *      12
+         *      \end{pmatrix}
+         *      /
+         *      4
+         *      \rightarrow
+         *      \begin{pmatrix}
+         *      16\\
+         *      12
+         *      \end{pmatrix}
+         *      \oslash
+         *      \begin{pmatrix}
+         *      4\\
+         *      4
+         *      \end{pmatrix}
+         *      =
+         *      \begin{pmatrix}
+         *      \tfrac{16}{4} \\
+         *      \tfrac{12}{4}
+         *      \end{pmatrix}
+         *      \rightarrow
+         *      \begin{pmatrix}
+         *      4\\
+         *      3
+         *      \end{pmatrix}
+         * \f]
+         * @param other The other scalar that acts as the divisor for each component of this vector.
+         * @return A GeoVecor2F who's components consist of the entrywise quotent of this and another scalar.
+         */
         GeoVector2F operator/(float other) const noexcept
         {
             return GeoVector2F(*reinterpret_cast<const glm::vec2*>(this) / other);
         }
 
+        /**
+         * @brief Constructs a GeoVector2F containing the entrywise sum of this and another GeoVector2F and applies it
+         * to this instance.
+         *
+         * @details
+         * An example of an entrywise addition using 2 dimensional vectors: \f[
+         *      \begin{pmatrix}
+         *      2\\
+         *      3
+         *      \end{pmatrix}
+         *      +
+         *      \begin{pmatrix}
+         *      4\\
+         *      5
+         *      \end{pmatrix}
+         *      =
+         *      \begin{pmatrix}
+         *      2 + 4\\
+         *      3 + 5
+         *      \end{pmatrix}
+         *      \rightarrow
+         *      \begin{pmatrix}
+         *      6\\
+         *      8
+         *      \end{pmatrix}
+         * \f]
+         *
+         * @param other The other vector who's components get added together with this vector.
+         * @return This GeoVecor2F who's components consist of the entrywise sum of this and another GeoVector2F.
+         */
         inline GeoVector2F& operator+=(GeoVector2F other) noexcept
         {
             *reinterpret_cast<glm::vec2*>(this) =
                 *reinterpret_cast<const glm::vec2*>(this) + *reinterpret_cast<const glm::vec2*>(&other);
             return *this;
         }
 
+        /**
+         * @brief Constructs a GeoVector2F containing the entrywise difference of this and another GeoVector2F and
```

Comment:
construct on operator. Also this is in-place memory overwriting as it returns `T&` to `this`.